feat(usePermission): add new hook (#88)
Elonsoft/elonkit-react: src/hooks/usePermission/usePermission.stories.mdx

import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
import LinkTo from '@storybook/addon-links/react';

import * as stories from './usePermission.stories';

<Meta title="Hooks/usePermission" />

# usePermission

> The side-effect hook to query permission status of browser APIs.

Reviewer: ```suggestion
The hook that queries permission status of browser APIs.
```
